From 0092f2352d30da684811306fb0ebfa76bcf339bc Mon Sep 17 00:00:00 2001
From: ysiyer <yegnesh.s.iyer@intel.com>
Date: Mon, 17 Nov 2014 16:44:25 +0530
Subject: [PATCH 1/1] Fixing compilation error if GPIOLIB is not enabled

gpio_chip structure is defined only if GPIOLIB is enabled.
This caused compilation errors while including the file
inte_soc_pmic.h in any of the source file. Hence added the
compile switch for defining the structre member of type gpio_chip.

Change-Id: If60ef9893a4e91745bdf0c2a2bdb24f8cbec3b21
Signed-off-by: ysiyer <yegnesh.s.iyer@intel.com>
---
 include/linux/mfd/intel_soc_pmic.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/mfd/intel_soc_pmic.h b/include/linux/mfd/intel_soc_pmic.h
index fbb5c75..c03626f 100644
--- a/include/linux/mfd/intel_soc_pmic.h
+++ b/include/linux/mfd/intel_soc_pmic.h
@@ -65,7 +65,9 @@ struct pmic_gpio_data {
 
 struct pmic_gpio {
 	struct mutex		buslock; /* irq_bus_lock */
+#ifdef CONFIG_GPIOLIB
 	struct gpio_chip	chip;
+#endif
 	int			irq;
 	int			irq_base;
 	int			update;
-- 
1.9.1

